Perspective recommendation faq updates (#3298)
harness/developer-hub: docs/faqs/cloud-cost-management-faqs.md

There is no restriction, but Harness recommends avoiding having the same connectors in FirstGen and NextGen.

#### Do we support moving the recommendations from Applied to Open recommendations section ? 

> No. Currently we support only moving the recommendations from Open to applied section as of now

Reviewer: ```suggestion
No. Currently, CCM supports only moving the recommendations from the Open to the Applied tab.
```

Author: Added the suggestion to the faq

#### Why are there no recommendations for GCP?

> We don’t support any GCP-specific recommendations as of today. We only have the support for the recommendations mentioned in the [doc](https://developer.harness.io/docs/category/recommendations)

Reviewer: ```suggestion
Currently, CCM doesn’t support any GCP-specific recommendations. For more information, go to [Recommendations](https://developer.harness.io/docs/category/recommendations).
```

Author: Added a slight modification to the text as mentioned below.
Currently, CCM does not offer any GCP-specific recommendations. Our support is limited to the recommendations outlined in the [documentation](https://developer.harness.io/docs/category/recommendations).

Reviewer: This is grammatically correct. But we usually refrain from using "We" and "Our" in public documentation. It's better to say "Support is limited to the recommendations specified in the documentation."

#### How long do we show the recommendations ? 

> For Workload, Node Pool and Azure VM recommendations we will keep showing the recommendation for 7 days and for EC2 and ECS recommendations it will be 15 days. This is only for the recommendation details page. On the recommendation listing page we only show the recommendations which were updated in last 4 days.

Reviewer: ```suggestion
For Workloads, Node pools, and Azure VMs, CCM provides recommendations for 7 days, while for EC2 and ECS, it provides recommendations for 15 days. Note that these durations pertain to the recommendations details page. On the recommendations listing page, CCM displays only the recommendations updated in the last 4 days.
```

Author: This is no longer valid so not adding it to the doc

## On-premises

> #### Does CCM support on-premises/Self Managed Platform (SMP) installations?

Reviewer: ```suggestion
#### Does CCM support on-premises A.K.A Self-Managed Platform (SMP) installations?
```

> Yes. CCM supports the following features and functionalities in SMP environment. Connectors set up is supported only on AWS and Kubernetes as of now. Also we support Perspectives, Budgets, Scheduled Reports, Cost categories, Anomalies and Recommendations for now.

Reviewer: ```suggestion
Yes. CCM supports the following features and functionalities in the SMP environment for AWS and Kubernetes:
- Connector setup 
- Perspectives, 
- Budgets
- Scheduled Reports
- Cost categories
- Anomalies 
- Recommendations 
```

Author: Added to the faq doc

  ![](./static/dashboard-time-filters.png)

> ### How costs are displayed if the perspective contains costs which aren't associated with the grouped by identity ?

Reviewer: ```suggestion
### How are costs displayed if the perspective contains costs that aren't associated with the _grouped by_ identity?
```

> We show `No` followed by `<group by>` for costs that don’t have any relation with the selected `<group by>`. 

Reviewer: ```suggestion
CCM displays `No` followed by the <group by>` element for costs that don’t have any relation with the selected `<group by>` element.  For more information, go to [Create Perspectives](https://developer.harness.io/docs/cloud-cost-management/use-ccm-cost-reporting/ccm-perspectives/create-cost-perspectives#review-no-accountprojectetc).
```

> Example : If a perspective contains rules for both AWS and GCP, and if the group by is done on GCP > SKU, the costs assosciated with other than GCP SKU will be displayed as No SKUs.

Reviewer: ```suggestion
For example, if a perspective contains rules for both AWS and GCP and if it is grouped by GCP > SKU, _No SKUs_ is displayed for costs (AWS, clusters, etc.) that don’t have any GCP SKUs associated with it.
```
